packages/domain/src/entities/Transactions.ts

```diff
@@ -119,7 +119,7 @@ export enum TransactionErrorReason {
 export class TransactionError extends Error {
   name = 'TransactionError' as const;
 
-  constructor(readonly reason: TransactionErrorReason) {
-    super(`Transaction failed due to: ${reason}`);
+  constructor(readonly reason: TransactionErrorReason, readonly txHash?: string) {
```

Comment:
I thought Cesare removed that in some other PR, I guess it's coming from a merge conflict.

Reply:
cool, removed it here again too